### ISSUE TITLE
Kitti Validation Results overfitting?

### ISSUE THREAD

**Issue by rancheng** [XinJCheng/CSPN#27]
Hi, Thanks for your contributions, as we have made our own branch to train it in Kitti ([# 25 pull request](https://github.com/XinJCheng/CSPN/pull/25)) and after one week training, we got the following results:

[val_result.txt](https://github.com/XinJCheng/CSPN/files/3708029/val_result.txt)

and the following results on validation samples, we made totally 2000+ inference on validation sets and found most of the samples are look like this:

![sample1](https://user-images.githubusercontent.com/2719771/66496857-8ee06300-ea89-11e9-8f78-f049751ab1eb.png)

**Comment by XinJCheng** [comment]
Thanks for your attention, how many sparse points have you sampled? 
We also find this pattern in our results, please check our ECCV paper, but we don't think this is the result of overfitting. We submitted a new paper to AAAI this year, these patterns have been overcome.

**Comment by longyangqi** [comment]
> Thanks for your attention, how many sparse points have you sampled?
> We also find this pattern in our results, please check our ECCV paper, but we don't think this is the result of overfitting. We submitted a new paper to AAAI this year, these patterns have been overcome.

Is the new paper available?

**Comment by ruiw0w** [comment]
Hello, thank you for providing the script for training the kitti dataset! What is the best result obtained by inferring the model you trained on the kitti data set on the validation set? I used your training script and added the log10 error indicator. After training, nan values ​​will appear and continue. Is this a phenomenon of over-fitting?
